src/main/java/org/scijava/parallel/DefaultParallelService.java

```diff
@@ -115,9 +115,9 @@ private String serializeProfile(
 	{
 		try {
 			final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-			final ObjectOutputStream oos = new ObjectOutputStream(baos);
-			oos.writeObject(profile);
-			oos.close();
+			try(final ObjectOutputStream oos = new ObjectOutputStream(baos)) {
```

Comment:
Good spot! Really like the performed change.

However, could you run automatic formatting in Eclipse?